wazimap/templates/homepage.html

```diff
@@ -102,7 +102,7 @@ <h1 class="article-header">About {{ WAZIMAP.name }}</h1>
             <p>Read more about <a href="/about">who we are and why we built Wazimap</a>.
 
             <h2 class="column-full subhead">Shoulders of Giants</h2>
-            <p>Wazimap is based on the amazing <a href="http://censusreporter.org/" target="_blank">Census Reporter</a> and is completely open source. You can <a href="https://github.com/Code4SA/censusreporter/">fork Wazimap on Github</a> and build one for your country.</p>
+            <p>Wazimap is based on the amazing <a href="http://censusreporter.org/" target="_blank">Census Reporter</a> and is completely open source. You can build one for your country by following the documentation <a href="https://wazimap.readthedocs.org/en/latest/index.html">here</a>.</p>
```

Comment:
I'm not a fan of hyperlinks that say "here". How about: "You can [build one for your country]."